editor/src/clj/editor/properties_view.clj

```diff
@@ -221,50 +221,61 @@
 
 (defn- handle-label-drag-event! [property-fn drag-update-fn update-ui-fn ^MouseDragEvent event]
   (.consume event)
-  (let [target (.getTarget event)
-        property (property-fn)
-        edit-type (:edit-type property)
-        to-fn (:to-type edit-type identity)
-        from-fn (:from-type edit-type identity)
-        min-val (:min edit-type)
-        max-val (:max edit-type)
-        x (.getX event)
-        y (.getY event)
-        [prev-x prev-y] (ui/user-data target ::position)
-        delta-x (- x prev-x)
-        delta-y (- prev-y y)
-        max-delta (if (> (abs delta-x) (abs delta-y)) delta-x delta-y)
-        update-val (cond-> (or (:precision edit-type) 1.0)
-                     (.isShiftDown event) (* 10.0)
-                     (.isControlDown event) (* 0.1)
-                     (neg? max-delta) -)]
-    (when (> (abs max-delta) 1)
-      (let [op-seq (ui/user-data target ::op-seq)
-            values (or (ui/user-data target ::values)
-                       (properties/values property))
-            new-values (mapv (fn [value]
-                               (cond-> (drag-update-fn value update-val)
-                                 min-val (max min-val)
-                                 max-val (min max-val))) values)]
-        (properties/set-values! property values op-seq)
-        (ui/user-data! target ::position [x y])
-        (ui/user-data! target ::values new-values)
-        (update-ui-fn (mapv (comp to-fn from-fn) new-values)
-                      (properties/validation-message property)
-                      (properties/read-only? property))))))
+  (let [property (property-fn)
+        target (.getTarget event)
+        position (ui/user-data target ::position)
+        op-seq (ui/user-data target ::op-seq)
+        is-read-only (properties/read-only? property)]
+    (when (and position op-seq (not is-read-only))
```

Comment:
This is the fix that is related to the mentioned issue. We somehow end up here without a position. I couldn't reproduce this, so I don't know how the drag event is triggered without going through a `MOUSE_PRESSED` event first, but I have experienced cases where the drag event is moved to a different target while dragging, so this is not surprising.

The read-only check was also moved here, because its state is dynamic, and we used to check for it upon the initialization of the property control. Changing a property that alters the read-only state of other props, would result in an inconsistent state.